src/expressServer.ts

```diff
@@ -120,7 +129,7 @@ function initializeRoutes(
 
   const app = express();
 
-  app.use(Sentry.Handlers.requestHandler());
+  app.use(Sentry.Handlers.requestHandler({user: false}));
```

Comment:
I assume this is to prevent the default of Sentry to try and extract user info from `req.user`?

Reply:
Correct, it tries to pull out .id, username, and email. We could also switch this to `user: ["_id"]` but I'd rather not risk leaking data (if something changes in Sentry, etc) and do it manually in the middleware.